hosts/hetzner-ephemeral-beta/default.nix

```diff
@@ -0,0 +1,76 @@
+{ pkgs, config, inputs, lib, ... }:
+{
+  # User options
+  users = {
+    juuso.authorizedKeys = [ "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNMKgTTpGSvPG4p8pRUWg1kqnP9zPKybTHQ0+Q/noY5+M6uOxkLy7FqUIEFUT9ZS/fflLlC/AlJsFBU212UzobA= ssh@secretive.sandbox.local" ];
+    kari.authorizedKeys = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIKEdpdbTOz0h9tVvkn13k1e8X7MnctH3zHRFmYWTbz9T kari@torque" ];
+  };
+
+  # Users group
+  users.groups.users = { };
+
+  # Localization
+  networking.hostName = "hetzner-ephemeral-beta";
+  time.timeZone = "Europe/Helsinki";
+
+  # Support for cross compilation
+  boot.binfmt.emulatedSystems = [
+    "armv7l-linux"
+  ];
+
+  # Hercules CI
+  services.hercules-ci-agent = {
+    enable = true;
+    settings.baseDirectory = "/var/mnt/.config/hercules-ci-agent";
+  };
+
+  systemd.mounts = [
+    {
+      enable = true;
+
+      what = "/dev/disk/by-label/nix";
+      where = "/var/mnt/.config";
+      type = "btrfs";
+      options = "subvolid=257";
+
+      before = [ "hercules-ci-agent.service" ];
+      wantedBy = [ "multi-user.target" ];
+    }
+  ];
+
+  systemd.services.nix-remount = {
+    path = [ "/run/wrappers" ];
+    enable = true;
+
+    serviceConfig = {
+      Type = "oneshot";
+    };
+
+    # if a new device, the new .rw-store has to be mounted
+    preStart = ''
+      /run/wrappers/bin/mount /dev/disk/by-label/nix -o subvolid=256 /nix/.rw-store
+    '';
+    script = ''
+      /run/wrappers/bin/mount -t overlay overlay -o lowerdir=/nix/.ro-store:/nix/store,upperdir=/nix/.rw-store/store,workdir=/nix/.rw-store/work /nix/store
+    '';
+
+    wantedBy = [ "multi-user.target" ];
+  };
+
+  # SSH
+  services.openssh = {
+    enable = true;
+    allowSFTP = false;
+    extraConfig = ''
+      AllowTcpForwarding yes
+      X11Forwarding no
+      AllowAgentForwarding no
```

Comment:
Also, please comment out `AllowAgentForwarding no` for now, this would be useful initially to share nix stores between build machines using the `nix copy` command.